src/test/java/uk/gov/hmcts/ccd/BaseTest.java

```diff
@@ -24,12 +23,11 @@
 public abstract class BaseTest {
 
     protected static final MediaType contentType = new MediaType(MediaType.APPLICATION_JSON.getType(),
-                                                                 MediaType.APPLICATION_JSON.getSubtype(),
-                                                                 Charset.forName("utf8"));
+        MediaType.APPLICATION_JSON.getSubtype(),
+        Charset.forName("utf8"));
     protected static final ObjectMapper mapper = new ObjectMapper();
 
     @Autowired
-    @Qualifier("EmbeddedPostgres")
```

Comment:
Hi Praveen what's the motivation for this change please?

Reply:
@mario-paniccia Sateesh added it, see here: https://github.com/hmcts/ccd-user-profile-api/pull/217#issuecomment-556345189